qiskit/qasm2/__init__.py

```diff
@@ -15,16 +15,18 @@
 OpenQASM 2 (:mod:`qiskit.qasm2`)
 ================================
 
-Qiskit has support for interoperation with OpenQASM 2.0 programs, both parsing into Qiskit formats
-and exporting back to OpenQASM 2.  The parsing components live in this module, while currently the
-export capabilities are limited to being the :meth:`.QuantumCircuit.qasm` method.
+Qiskit has support for interoperation with OpenQASM 2.0 programs, both :ref:`parsing into Qiskit
+formats <qasm2-parse>` and :ref:`exporting back to OpenQASM 2 <qasm2-export>`.
 
 .. note::
 
     OpenQASM 2 is a simple language, and not suitable for general serialisation of Qiskit objects.
```

Comment:
```suggestion
    OpenQASM 2 is a simple language, and not suitable for general serialisation of Qiskit circuits.
```

Reply:
I'd done this deliberately because QPY also allows serialisation of `ScheduleBlock`, and sometimes the circuit structure is serialisable, but one or two of the contained instructions / expressions aren't.  I can change it if you think that distinction is too subtle, though.